src/app/auth/auth.js

```diff
@@ -0,0 +1,71 @@
+$(document).ready((function ($) {
+    'use strict';
+
+    function getFormData($form) {
```

Comment:
`getFormData` is a duplicate of the same function in `home.js`, move this function to `app.js` and make it globally available through `window.getFormData`, replace all places using `getFormData` with `window.getFormData`

Reply:
Done

Reply:
Resolved.